src/content/app/tools/blast/views/blast-submission-results/components/blast-results-per-sequence/BlastResultsPerSequence.tsx

```diff
@@ -61,18 +62,24 @@ const BlastResultsPerSequence = (props: BlastResultsPerSequenceProps) => {
   const { width: plotwidth } = useResizeObserver({ ref: rulerContainer });
   const [shouldShowJobResult, showJobResult] = useState(true);
   const [shouldShowParamaters, showParamaters] = useState(false);
+  const usesGenomicHitsDiagram = parameters.database === 'dna'; // NOTE: works for now; but likely to expand in the future
```

Comment:
There is `needsGenomicHitsDiagram` used in `SingleBlastJobResult`. Can it be used here too instead of `usesGenomicHitsDiagram`?

Reply:
I didn't particularly like that name; but I suppose `uses...` may me too similar with names of hooks? 🤷‍♂️  I renamed the variable to `needs...`